data/src/main/java/com/microsoft/azure/kusto/data/Client.java

```diff
@@ -7,10 +7,15 @@
 import com.microsoft.azure.kusto.data.exceptions.DataServiceException;
 
 public interface Client {
-
     KustoOperationResult execute(String command) throws DataServiceException, DataClientException;
 
     KustoOperationResult execute(String database, String command) throws DataServiceException, DataClientException;
 
     KustoOperationResult execute(String database, String command, ClientRequestProperties properties) throws DataServiceException, DataClientException;
-}
+
+    String executeForJsonResult(String database) throws DataServiceException, DataClientException;
```

Comment:
Naming this was tough. On the one hand it's uncommon to name a method to describe its return format. Worse, here we both do and have no choice but to provide a name that's inconsistent with the other names.

An alternative was "executeReturnJson".

Reply:
@vladikbr suggested "executeWithJsonResult"

Reply:
I think that executeWithJsonResult is the best option here, with executeForJsonResult at close second

Reply:
I think that "with" implies the json is an input parameter, and that "for" implies the json is an output parameter (which is the case).

I'm outvoted, so I'll change before submitting unless someone agrees with this logic first.

Reply:
How about "executeToJsonResult"?

Reply:
@vladikbr @AsafMah @ohadbitt 
Per Sheldon's confirmation a couple hours ago that he'd like this API as it is here, I'd like to conclude on the proper naming so we can merge this.

My order of preference is:
1) executeToJsonResult
2) executeForJsonResult
3) executeWithJsonResult

Reply:
To is fine